stdlib/area/area.lua

```diff
@@ -314,4 +330,28 @@ function Area.tostring(area)
     return 'Area {'..left_top..', '..right_bottom ..'}'
 end
 
+--- Converts an entity and its collision_box to the area around it
+-- @see Entity.to_collision_area
+-- @tparam LuaEntity entity
+-- @treturn LuaBoundingBox
+function Area.to_collision_area(entity)
+    fail_if_missing(entity, "missing entity argument")
+
+    local pos = entity.position
+    local bb = entity.prototype.collision_box
+    return Area.offset(bb, pos)
+end
+
+--- Converts an entity and its selection_box to the area around it
+-- @see Entity.to_selection_area
+-- @tparam LuaEntity entity
+-- @treturn LuaBoundingBox
+function Area.to_selection_area(entity)
+    fail_if_missing(entity, "missing entity argument")
```

Comment:
This is a duplicate of https://github.com/Afforess/Factorio-Stdlib/blob/master/stdlib/entity/entity.lua#L13